game-core/src/main/java/games/strategy/engine/pbem/PbemMessagePoster.java

```diff
@@ -27,13 +27,14 @@
  * although the PBEM games will always be local
  */
 @Log
-public class PBEMMessagePoster implements Serializable {
+public class PbemMessagePoster implements Serializable {
   public static final String FORUM_POSTER_PROP_NAME = "games.strategy.engine.pbem.IForumPoster";
   public static final String EMAIL_SENDER_PROP_NAME = "games.strategy.engine.pbem.IEmailSender";
-  public static final String PBEM_GAME_PROP_NAME = "games.strategy.engine.pbem.PBEMMessagePoster";
+  public static final String PBEM_GAME_PROP_NAME = "games.strategy.engine.pbem.PbemMessagePoster";
```

Comment:
I believe it's okay to change this property name since we are breaking save game compatibility.